backend/src/main/java/saci/application/RoleController.java

```diff
@@ -40,8 +42,9 @@ public class RoleController {
             })
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
-    public Role createRole(@RequestBody Role role) {
-        return roleService.createRole(role);
+    public ResponseEntity<Role> createRole(@RequestBody Role role) {
+        Role createdRole = roleService.createRole(role);
+        return createdRole == null ?   ResponseEntity.status(HttpStatus.CONFLICT).build() : ResponseEntity.ok(createdRole);
```

Comment:
Run in the backend folder `mvn spotless:apply` to take out these empty spaces in the code (Spotless also format the code, like Prettier in frontend)